### DISCUSSION TITLE
Connect with my local database

### DISCUSSION THREAD

**Discussion by abdullahameromran** [supabase/supabase#26367] [Questions]
How to use supabase to connect with local database and all aspects about that ?

**Comment by silentworks** [comment]
Your question isn't very clear here. Can you rephrase and clarify what you are asking here please?

**Comment by abdullahameromran** [comment]
Thanks for your reply

I have confusion here for how to run Supabase on my oracle database or any
database local

Does when I run Supabase dashboard locally in my machine I have the option
to store locally or just same like online dashboard?

**Reply by silentworks** [reply]
Supabase is built on top of Postgres, so you wouldn't be able to use it with oracle database. You can run a local instance of supabase using its CLI if you need to develop locally first. You can read more here https://supabase.com/docs/guides/cli
